<launch>
  <node pkg="disparity_to_point_cloud" type="depth_map_fusion_node" name="depth_map_fusion" output="screen">
    <remap to="/uvc_camera/cam_0/image_raw" from="/astra/depth/image_raw "/>
    <remap to="/uvc_camera/cam_1/image_raw" from="/astra/depth/image"/>
    <remap to="/uvc_camera/cam_2/image_raw" from="/astra/depth_registered/points"/>
    <remap to="/uvc_camera/cam_3/image_raw" from="/astra/depth/points"/>

    <param name="offset_x" value="-7" />
    <param name="offset_y" value="15" />
  </node>
</launch>
